src/index.js

```diff
@@ -10,6 +10,7 @@ if (get(window, versionField, null)) {
 
 set(window, versionField, 'v<?= PACKAGE(version) ?>');
 
+export * from './bidders';
```

Comment:
It turns out that we don't need to concat bidders to global-bundle. Am I right? https://github.com/Wikia/ad-products/blob/dev/maintenance/build-bundle.sh#L4

Reply:
I think you right. Removing.